### PR TITLE
feat(inline-message): angular wrapper for inline message

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

Angular wrapper for inline-message component.

-----

This PR requires a functional validation from @jeremie-lucca in order to make sure the icon behavior is the one we want to have.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2212/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2212/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2212/storybook

File: packages/ng/inline-message/inline-message.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@Component({
+	selector: 'lu-inline-message',
+	standalone: true,
+	imports: [NgIf],
+	templateUrl: './inline-message.component.html',
+	styleUrls: ['./inline-message.component.scss'],
+})
+export class InlineMessageComponent {
+	@Input({ required: true })
```
Comment (GuillaumeNury) [inline packages/ng/inline-message/inline-message.component.ts:12]: 🚀

**Review by GuillaumeNury — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**